crm_action/__openerp__.py

```diff
@@ -22,17 +22,14 @@
 
 {
     'name': 'CRM Action',
-    'version': '8.0.1.1.0',
-    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'version': '9.0.1.1.0',
+    'author': 'Savoir-faire Linux,Odoo Community Association (OCA),AtulArvind',
```

Comment:
These are the original authors. Keep the original ones unless significant original changes have been made.